components/clarinet-format/tests/golden.rs

```diff
@@ -47,6 +47,7 @@ fn format_file_with_metadata(source: &str) -> String {
     formatter.format_file(&real_source)
 }
 #[test]
+#[ignore]
```

Comment:
to remove before merge

Reply:
You can leave it in. I'll remove it with my perf branch soon